apps/meteor/tests/data/users.helper.js

```diff
@@ -90,3 +91,18 @@ export const setUserStatus = (overrideCredentials = credentials, status = UserSt
 		message: '',
 		status,
 	});
+
+export const registerUser = (userData = {}, overrideCredentials = credentials) => new Promise((resolve) => {
```

Comment:
this can be `async`:
```js
export const registerUser = async (userData = {}, overrideCredentials = credentials) => {
	const result = await request
		.post(api('users.register'))
                 // ...

	return result.body.user;
};
```